### ISSUE TITLE
Better control over when GC runs

### ISSUE THREAD

**Issue by FeldrinH** [Manishearth/rust-gc#169]
Currently it seems that GC can trigger a collection any time a new Gc<T> is allocated. I would like to have manual control over when collection can happen. (I mean I want to prevent automatic collections during allocation from happening and instead manually call a function that checks the threshold and collects if there is too much allocated memory.)

**Comment by stephanemagnenat** [comment]
Yes that would be nice to have. For example in a scripting language used in a game, one would like to run the garbage collection in between frames (e.g., after the frame has been rendered and sent to the display).
